crates/networking/p2p/rlpx/connection.rs

```diff
@@ -606,12 +597,16 @@ impl<S: AsyncWrite + AsyncRead + std::marker::Unpin> RLPxConnection<S> {
     }
 
     async fn receive_handshake_msg(&mut self) -> Result<Vec<u8>, RLPxError> {
-        let mut buf = vec![0; MAX_DISC_PACKET_SIZE];
+        let mut buf = vec![0; 2];
 
         // Read the message's size
         self.framed.get_mut().read_exact(&mut buf[..2]).await?;
```

Comment:
Now that buf is exactly 2 bytes size, the index is not necessary
```suggestion
        self.framed.get_mut().read_exact(&mut buf).await?;
```

Reply:
Solved in [d6532fc](https://github.com/lambdaclass/ethrex/pull/1768/commits/d6532fc292750e287118736e973a68b78d31c0aa).